actors/builtin/reward/reward_state.go

```diff
@@ -9,6 +9,12 @@ import (
 // A quantity of space * time (in byte-epochs) representing power committed to the network for some duration.
 type Spacetime = big.Int
 
+// 36.242212 FIL
+var InitialRewardPositionEstimate = big.Mul(abi.NewTokenAmount(36242212), abi.NewTokenAmount(1e12))
+
+// -1.0982489*10^-7 FIL per epoch.  Change of simple minted tokens between epochs 0 and 1
```

Comment:
https://www.wolframalpha.com/input/?i=e%5E-%28Log%5B2%5D%2F%286+*+60*60*365*24%2F+30+%29%29-1

and this should be the value for initial velocity estimate